Compilation error: temporary value dropped while borrowed

```
Checking debian-control v0.1.17
error[E0716]: temporary value dropped while borrowed
   --> <...>/debian-control-0.1.17/src/changes.rs:152:14
    |
149 |         let subdir = if source.starts_with("lib") {
    |             ------ borrow later stored here
...
152 |             &source.chars().next().unwrap().to_string()
    |              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ creates a temporary value which is freed while still in use
153 |         };
    |         - temporary value is freed at the end of this statement
    |
    = note: consider using a `let` binding to create a longer lived value

For more information about this error, try `rustc --explain E0716`.
error: could not compile `debian-control` (lib) due to 1 previous error
```

--- a/debian-control/src/changes.rs
+++ b/debian-control/src/changes.rs
@@ -147,9 +147,9 @@ impl Changes {
         let source = self.source()?;
 
         let subdir = if source.starts_with("lib") {
-            "lib"
+            "lib".to_string()
         } else {
-            &source.chars().next().unwrap().to_string()
+            source[..1].to_lowercase()
         };
 
         Some(format!("pool/{}/{}/{}", section, subdir, source))

